src/mapping/mapping_helpers.py

```diff
@@ -2,47 +2,116 @@
 import pandas as pd
 import logging
 
-from src.utils.wrappers import time_logger_wrap, exception_wrap
-
 MappingLogger = logging.getLogger(__name__)
 
 
-def validate_ultfoc_df(df: pd.DataFrame) -> pd.DataFrame:
+def col_validation_checks(
+    df: pd.DataFrame,
+    mapper_name: str,
+    col: str,
+    expected_value_type: type,
+    expected_value_length: int,
```

Comment:
It might be useful to have "None" as the default for expected_value_type and _length, otherwise people might not know that's what you can pass.

Reply:
I have added those